.github/workflows/test_backend_uv.yaml

```diff
@@ -27,12 +27,8 @@ jobs:
         run: uvx ruff check --config ruff.toml
         continue-on-error: false
 
-      - name: Start backend
-        run: make start-lumigator-build
+      - name: Run unit tests
```

Comment:
non-blocking nit: call this "Run backend unit tests" for clarity, and same for integration tests and SDK unit/integration tests

Reply:
Wilco